src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/LanguageClient/Cohost/CohostSignatureHelpEndpoint.cs

```diff
@@ -17,7 +17,9 @@
 using Microsoft.VisualStudio.Razor.LanguageClient.Cohost;
 using Microsoft.VisualStudio.Razor.LanguageClient.Extensions;
 using Microsoft.VisualStudio.Razor.Settings;
-using RLSP = Roslyn.LanguageServer.Protocol;
+using static Roslyn.LanguageServer.Protocol.RoslynLspExtensions;
+using RoslynLspFactory = Roslyn.LanguageServer.Protocol.RoslynLspFactory;
```

Comment:
This is surprising

Reply:
It was at one point while I was working on this, but it doesn't look like it's needed any longer. Good cathc!

Reply:
Oops, it is still needed. There's a call to `RoslynLspFactory.CreatePosition(request.Position.ToLinePosition())` below.

Reply:
Right but where else is `RoslynLspFactory` used that the alias is needed to disambiguate?